### ISSUE TITLE
Dataset features description

### ISSUE THREAD

**Issue by vfdev-5** [ramp-kits/autism#18]
Hi,
I was just wondering whether somewhere there is a more detailed and structured description of data features?  For example, what is `participants_site` ?
Thanks

**Comment by alirezamosh** [comment]
I second that. Domain knowledge would be very helpful for this challenge as there are people that are not familiar with the description of features.

**Comment by glemaitre** [comment]
Participants site are the hospital where the acquisition have been done.

**Comment by glemaitre** [comment]
@alirezamosh check also the website. If you have more questions we can discuss in more details.

**Comment by vfdev-5** [comment]
@glemaitre concerning "anatomy" features [this site](http://surfer.nmr.mgh.harvard.edu/fswiki/MorphometryStats) has been already sited in one of the issues. Maybe it would good to include it as a reference for those who is not famiiliar with the domain.

**Comment by glemaitre** [comment]
I see.

@GaelVaroquaux Do you think that it should be added in a sort of introduction in the notebook or shall we have a wiki or an aside page info for those features. I am scared to clutter the notebook by adding up information.

**Comment by vfdev-5** [comment]
> I am scared to clutter the notebook by adding up information.
This information can be stored in a additional file, for example

**Comment by glemaitre** [comment]
> This information can be stored in a additional file, for example

Yep I had in mind the following suggestion:

- an additional file.
- an additional webpage on the main site.
- on the wiki page of the repository.

**Comment by vfdev-5** [comment]
Another question for brain anatomy specialists, according to [wiki](https://en.wikipedia.org/wiki/Ventricular_system) it seems there are 4 ventricles: 
- lateral ventricles right and left (one for each hemisphere)
- third ventricle
- fourth ventricle

However in the data we have also `anatomy_5th-Ventricle` column with data like: 

![image](https://user-images.githubusercontent.com/2459423/39971143-b4086f24-56f6-11e8-9841-b5971e52bf7e.png)

**Comment by glemaitre** [comment]
@vfdev-5 You are right. The extraction was done with FreeSurfer so you will find information there regarding the anatomy.

**Comment by glemaitre** [comment]
@vfdev-5 you can let this issue opened. This is actually useful and this is not a duplicate.

**Comment by suprsankr** [comment]
I am wondering what the following anatomical features mean (are they thickness or area or volume measures ?)

- anatomy_Left-Lateral-Ventricle
-  anatomy_Left-Inf-Lat-Vent
-  anatomy_Left-Cerebellum-White-Matter
-  anatomy_Left-Cerebellum-Cortex
-  anatomy_Left-Thalamus-Proper
-  anatomy_Left-Caudate
-  anatomy_Left-Putamen
-  anatomy_Left-Pallidum
-  anatomy_3rd-Ventricle
-  anatomy_4th-Ventricle
-  anatomy_Brain-Stem
-  anatomy_Left-Hippocampus
-  anatomy_Left-Amygdala
-  anatomy_CSF
-  anatomy_Left-Lateral-Ventricle
-  anatomy_Left-Inf-Lat-Vent
-  anatomy_Left-Cerebellum-White-Matter
-  anatomy_Left-Cerebellum-Cortex
-  anatomy_Left-Thalamus-Proper
-  anatomy_Left-Caudate
-  anatomy_Left-Putamen
-  anatomy_Left-Pallidum
-  anatomy_3rd-Ventricle
-  anatomy_4th-Ventricle
-  anatomy_Brain-Stem
-  anatomy_Left-Hippocampus
-  anatomy_Left-Amygdala
-  anatomy_CSF
-  anatomy_Left-VentralDC
-  anatomy_Left-vessel
-  anatomy_Left-choroid-plexus
-  anatomy_Right-Lateral-Ventricle
-  anatomy_Right-Inf-Lat-Vent
-  anatomy_Right-Cerebellum-White-Matter
-  anatomy_Right-Cerebellum-Cortex
-  anatomy_Right-Thalamus-Proper
-  anatomy_Right-Caudate
-  anatomy_Right-Putamen
-  anatomy_Right-Pallidum
-  anatomy_Right-Hippocampus
-  anatomy_Right-Amygdala
-  anatomy_Right-VentralDC
-  anatomy_Right-vessel
-  anatomy_Right-choroid-plexus
-  anatomy_5th-Ventricle
-  anatomy_WM-hypointensities
-  anatomy_Left-WM-hypointensities
-  anatomy_Right-WM-hypointensities
-  anatomy_non-WM-hypointensities
-  anatomy_Left-non-WM-hypointensities
-  anatomy_Right-non-WM-hypointensities
-  anatomy_Optic-Chiasm
-  anatomy_CC_Posterior
-  anatomy_CC_Mid_Posterior
-  anatomy_CC_Central
-  anatomy_CC_Mid_Anterior
-  anatomy_CC_Anterior

**Comment by vfdev-5** [comment]
For more info on fsMRI, connectivity I found [this article](https://arxiv.org/pdf/1304.3880.pdf) and [this](http://nilearn.github.io/connectivity/index.html) somehow helpful. 

HTH

**Comment by romainVala** [comment]
@suprsankr 
those measures are volume because they  are not in the cortex (only cortical area are defined between 2 surface which allow the quantification of mean thickness and area)

**Comment by suprsankr** [comment]
@romainVala thank you! i did not quite understand the second part of the answer. The above measures are also volume measures (such as anatomy_Optic-Chiasm..) except that they are not part of the cortex. So these regions cannot conveniently be measured for a thickness and area measure. 
However for a non brain scientist, it suffices to know that the above measures are just volume measures. correct ? :)

**Comment by romainVala** [comment]
yes
